iso4217.go

```diff
@@ -15,11 +15,12 @@
 // specific language governing permissions and limitations
 // under the License.
 
-// Generated on 2023-06-02T09:32:24Z by adam, any modifications will be overwritten
+// Generated on 2023-06-05T04:30:46Z by raphaelsantodomingo, any modifications will be overwritten
 package iso4217
 
 type CurrencyCode struct {
-	Code, Name string
+	Code, Name    string
+	DecimalPlaces int
```

Comment:
Let's mention that `0` can be returned for some types, but that a non-negative value will be returned. We could use uint, sorry for the two changes here.

Reply:
all good, updating